resources/queries/cds/ds_visittag.sql

```diff
@@ -18,13 +18,9 @@ DISTINCT arm_visit.study_arm_visit_type AS Name,
 arm_visit.study_arm_visit_type AS caption,
 FALSE AS SingleUse
 FROM cds.import_studypartgrouparmvisit AS arm_visit
-WHERE study_arm_visit_type IS NOT NULL
+WHERE study_arm_visit_type IS NOT NULL AND study_arm_visit_type != 'Placeholder'
 
-UNION
-
-SELECT
-DISTINCT arm_visit.study_arm_visit_label AS Name,
-arm_visit.study_arm_visit_label AS caption,
-TRUE AS SingleUse
-FROM cds.import_studypartgrouparmvisit AS arm_visit
-WHERE study_arm_visit_label IS NOT NULL
\ No newline at end of file
+-- add the pre-defined alignment visits tags
+UNION ALL SELECT 'Enrollment' AS Name, 'Enrollment' AS caption, TRUE AS SingleUse
```

Comment:
Is the purpose of 'UNION ALL' to fail the ETL should any of 'Enrollment', 'First Vaccination' or 'Last Vaccination' is accidentally used in StudyPartGroupArmVisit.study_arn_visit_type?